src/display/editor/editor.js

```diff
@@ -627,11 +618,6 @@ class AnnotationEditor {
     if (!this.#keepAspectRatio) {
       this.div.style.height = `${((100 * height) / parentHeight).toFixed(2)}%`;
     }
-    this.#altTextButton?.classList.toggle(
-      "small",
-      width < AnnotationEditor.SMALL_EDITOR_SIZE ||
-        height < AnnotationEditor.SMALL_EDITOR_SIZE
-    );
```

Comment:
Can we also remove this code now?
https://github.com/mozilla/pdf.js/blob/d679078beb45314aa45f12c212ea9e21d3f3154f/src/display/editor/editor.js#L98-L100